src/structures/gdt.rs

```diff
@@ -193,7 +193,6 @@ impl<const MAX: usize> GlobalDescriptorTable<MAX> {
     ///
     /// Panics if the GDT doesn't have enough free entries.
     #[inline]
-    #[cfg_attr(feature = "const_fn", rustversion::attr(all(), const))]
```

Comment:
This removes the const qualifier from the function. We only want to remove the feature gate, `append` should now always be `const`.

The same applies to `push`.

Reply:
Whoops, thanks a lot!